### DISCUSSION TITLE
How do you handle multiple versions of a single release (e.g. Vinyl and CD-Rip)

### DISCUSSION THREAD

**Discussion by Popkornium18** [MusicPlayerDaemon/MPD#1479] [Q&A]
I would like to have multiple versions of my favorite albums in my collection. Sometimes there are some bonus tracks on CDs, but the Vinyl version uses a more dynamic master. Remasters of old albums would be another reason for something like this. How would you handle something like this with mpd?

The only solution I found was appending the release info to the album title like this: `Abbey Road [Vinyl; 1969; Apple Records (PCS 7088)]` and `Abbey Road [CD; 1987; Parlophone (C2 46446); Stereo Remaster]`. When done this way, mpd does not group both releases into one Album. I don't really like this solution as it breaks scrobbling with last.fm and abuses the Album title for something that is more or less only a comment.

Ideally I'm looking for something like this: 

1. Grouping albums by directory
2. A client that displays the `comment` tag, where I'd put the release info

Is something like that possible? Does someone have a different solution from what I came up with?

**Comment by MaxKellermann** [comment]
I ... don't understand the problem. Just add multiple copies of one album to your database. You did not describe what problem remains to be solved.

> Grouping albums by directory

What does that mean? A client can specify which subdirectory shall be searched. Do you want that?

> A client that displays the comment tag, where I'd put the release info

ncmpc displays the comment tag.

Anyway, putting a recording year into the album name doesn't make any sense. This belongs in the "date" tag. Once you have that, you can group by album name and date.

**Comment by Popkornium18** [comment]
> I ... don't understand the problem. Just add multiple copies of one album to your database. You did not describe what problem remains to be solved.
> What does that mean? A client can specify which subdirectory shall be searched. Do you want that?

I have some Albums where I have a vinyl rip, because it uses a nicer master and a japanese CD rip, because those often contain bonus tracks. Both of them were released in the same year so the tags are completely identical other than the comment tag. In cases like this it would be nice if they were displayed as two separate albums. But I guess there are valid use cases for having files of a single album in different folders and wanting them to be displayed as one in an MPD client.

**Reply by MaxKellermann** [reply]
> displayed as two separate albums

Displaying is purely a client thing, not a MPD thing. MPD gives clients all the tools to query and group everything.
